lib/vmdb/console_methods.rb

```diff
@@ -28,6 +28,7 @@ def simulate_queue_worker(break_on_complete = false)
         else
           break_on_complete ? break : sleep(1.second)
```

Comment:
I didn't know how many messages to specify, that is why I said - over estimate (anti price is right style)